.vscode/extensions.json

```diff
@@ -1,6 +1,5 @@
 {
 	"recommendations": [
-		"gbasood.byond-dm-language-support",
-		"platymuus.dm-langclient"
+		"ss13.byond"
```

Comment:
this is bundled extension with the same extensions as above, plus extension for commenting blocks